src/app/modules/main/wallet_section/view.nim

```diff
@@ -266,4 +267,23 @@ QtObject:
     self.delegate.resetRpcStats()
 
   proc canProfileProveOwnershipOfProvidedAddresses*(self: View, addresses: string): bool {.slot.} =
-    return self.delegate.canProfileProveOwnershipOfProvidedAddresses(addresses)
\ No newline at end of file
+    return self.delegate.canProfileProveOwnershipOfProvidedAddresses(addresses)
+
+  proc reloadAccountTokens*(self: View) {.slot.} =
+    self.delegate.reloadAccountTokens()
+
+  proc lastReloadTimestampChanged*(self: View) {.signal.}
+  proc setLastReloadTimestamp*(self: View, lastReloadTimestamp: int64) =
+    self.lastReloadTimestamp = lastReloadTimestamp
+    self.lastReloadTimestampChanged()
+  proc getLastReloadTimestamp(self: View): QVariant {.slot.} =
+    return newQVariant(self.lastReloadTimestamp)
+  QtProperty[QVariant] lastReloadTimestamp:
+    read = getLastReloadTimestamp
+    notify = lastReloadTimestampChanged
+
+
+  proc tokensReloaded*(self: View) {.signal.}
+  proc emitTokensReloaded*(self: View, timestamp: int64) =
+    self.setLastReloadTimestamp(timestamp)
```

Comment:
I would remove this line from there and add it to module, cause module should keep the logic.